File: backend-module/src/main/java/com/campus/backend/eneity/RiskWarning.java
package com.campus.backend.eneity;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class RiskWarning {
    private int id;
    private int studentId;
    private String semester;
    private String riskLevel;
    private String riskReason;
    private String aiAnalysis;
    private String handleStatus;
    private int handlerId;
    private String handleRemark;
    private LocalDateTime handleAt;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private boolean isDeleted;
}

File: frontend/src/types/riskWarning.ts
export interface RiskWarning {
  id: number
  studentId: number
  semester: string
  riskLevel: string
  riskReason: string | null
  aiAnalysis: string | null
  handleStatus: string
  handlerId: number
  handleRemark: string | null
  handleAt: string | null
  createdAt: string
  updatedAt: string
  isDeleted: boolean
}

export interface RiskWarningHandleForm {
  handleRemark: string
}
